index.html

```diff
@@ -151,8 +152,33 @@ <h2 class="myself">Hello I’m a software developer!
                     <li class="knowledge third_knowledge">Skills<img src="images/rightarrow.png" alt="right arrow"></li>
                     <hr>
                 </ul>
-            </div>
+            
     </section>
+
+    <!-- contact form -->
+    <form action="https://formspree.io/f/mqkoaqwr"
+                        method="POST" class="contact_form">
+        <h1 class="contact_me">Contact me</h1>
+        <h2 class="contact_details">
+            If you have an application you are interested in developing, 
+            a feature that you need built or a project that needs coding. 
+            I'd love to help with it.
+        </h2>
+        <ul class="list_items">
+            <li>
+                <input class="name" type="text" maxlength="30" required placeholder="Your name here">
+            </li>
+            <li>
+                <input class="email" type="email" id="email" size="30" required placeholder="Your mail here">
+            </li>
+            <li>
+            
+                <textarea class="message" name="message" id="" cols="30" rows="10" maxlength="500" required
+                    placeholder="Write your message here"></textarea>
+            </li>
+        </ul>
```

Comment:
I suggest you remove the inputs from the list element. A form is a container that holds various input elements. The way you have done it here confuses the screen readers. 
You can have them as 
`<form>`
`<input class="name" type="text" maxlength="30" required placeholder="Your name here"/>`
`<input class="email" type="email" id="email" size="30" required placeholder="Your mail here">`
`<textarea class="message" name="message" id="" cols="30" rows="10" maxlength="500" required
                    placeholder="Write your message here"></textarea>`
`</form>`

Reply:
Hi @Nyame-Wolf , thanks for your feedback! Its solved now 💯 
I did it that way because it was an indication in [one article](https://developer.mozilla.org/en-US/docs/Learn/Forms/Your_first_form) of the lessons.